src/Uno.UI/UI/Xaml/UIElement.Pointers.Managed.cs

```diff
@@ -71,7 +71,7 @@ private object CoerceHitTestVisibility(object baseValue)
 
 			// If we're not locally hit-test visible, visible, or enabled, we should be collapsed. Our children will be collapsed as well.
 			if (
-#if !__MACOS__
+#if !__MACOS__ && !__ANDROID__
```

Comment:
If you have to add this I think this is because this `HitTestVisibility` property coercing method is not re-evaluated on load. An unloaded element should be considered as `Collapsed`.

Reply:
I added this because it seems there is no `UIElement.IsLoaded` on Android at all.